docusaurus/docs/React/guides/date-time-formatting.mdx

```diff
@@ -0,0 +1,205 @@
+---
+id: date-time-formatting
+title: Date and time formatting
+keywords: [date, time, datetime, timestamp, format, formatting]
+---
+
+In this guide we will learn how date a time formatting can be customized within SDK's components.
+
+## SDK components displaying date & time
+
+The following components provided by the SDK display datetime:
+
+- `DateSeparator`- component separating groups of messages in message lists
+- `EventComponent` - component that renders system messages (`message.type === 'system'`)
+- `Timestamp` - component to display non-system message timestamp
+
+## Format customization
+
+The datetime format customization can be done on multiple levels:
+
+1. Override the default component prop values
+2. Supply custom formatting function
+3. Format date via i18n
+
+### Override the component props defaults
+
+All the mentioned components accept timestamp formatter props:
+
+```ts
+export type TimestampFormatterOptions = {
+  /* If true, call the `Day.js` calendar function to get the date string to display (e.g. "Yesterday at 3:58 PM"). */
+  calendar?: boolean | null;
```

Comment:
```suggestion
  enableCalendar?: boolean | null;
```
Can we rename this or would that be a breaking change?

Reply:
`calendar` prop existed before this PR, so probably a breaking change